src/operationbot/config.py

```diff
@@ -2,23 +2,23 @@
 from typing import Dict, List
 from zoneinfo import ZoneInfo
 
-import secret
+from operationbot import secret
 
 VERSION = 12
 # PURGE_ON_CONNECT = False
-_test_channel = 530411066585382912
+_test_channel = 1049819136072626186
 if secret.DEBUG:
     EVENT_CHANNEL = _test_channel
     EVENT_ARCHIVE_CHANNEL = _test_channel
     COMMAND_CHANNEL = _test_channel
     LOG_CHANNEL = _test_channel
     GAME = 'with bugs'
-    EMOJI_GUILD = 219564389462704130
+    EMOJI_GUILD = 1049808774308831332
 else:
-    EVENT_CHANNEL = 502824760036818964
-    EVENT_ARCHIVE_CHANNEL = 528914471700267029
-    COMMAND_CHANNEL = 528980590930821131
-    LOG_CHANNEL = 621066917339201547
+    EVENT_CHANNEL = 1049808836212564038
```

Comment:
These are my personal test server's values.
Ideally none of these should be in (py) code (instead config files), but I'll change it in a separate PR.